client/landing/stepper/declarative-flow/assembler.ts

```diff
@@ -0,0 +1,53 @@
+import { useFlowProgress, ASSEMBLER_FLOW } from '@automattic/onboarding';
+import { useDispatch } from '@wordpress/data';
+import { useSiteSlug } from '../hooks/use-site-slug';
+import { ONBOARD_STORE } from '../stores';
+import { recordSubmitStep } from './internals/analytics/record-submit-step';
+import PatternAssembler from './internals/steps-repository/pattern-assembler';
+import Processing from './internals/steps-repository/processing-step';
+import { Flow, ProvidedDependencies } from './internals/types';
+
+const assembler: Flow = {
```

Comment:
I'd prefer to call it `siteAssembler` as `assembler` is a bit ambiguous. WDYT?

Reply:
I'll make sure to name it `siteAssemblerFlow`, similar to `siteSetupFlow`, does it make sense?

Reply:
Sure!

Reply:
Update the file name as suggested and change the flow name into `site-assembler`